action.yml

```
@@ -13,6 +13,13 @@ inputs:
   GITHUB_TOKEN:
     description: 'A GitHub token.'
     required: true
+  ENABLE_MERGE_INFO_PREVIEW:
```

```suggestion
  ENABLE_GITHUB_API_PREVIEW:
```

In case we'll use other features from the API in the future?